lib/containers/urls.pm

```diff
@@ -293,6 +294,16 @@ sub get_suse_container_urls {
     return (\@untested_images, \@released_images);
 }
 
+# Get single image from CONTAINER_IMAGE_TO_TEST or untested image from get_suse_container_urls()
+sub get_container_url_to_test {
```

Comment:
```suggestion
sub get_container_url_to_test {
sub get_container_image_to_test {
```

Maybe we could align the naming of the function with the naming of the variable? WDYT?